stylesheet.css

```diff
@@ -1,146 +1,155 @@
+  position: relative;
 }
 
 .bannerImage {
-    position: relative;
-    display: block;
-    width: 1300px;
-    border-radius: 48px;
+  width: 100%;
+  border-radius: 48px;
 }
 
-/* Aleksandra Georgievska Line Header text */
-.AGHeaderPosition {
-    position: relative;
-    display: block;
-    width: 700px;
-    top: -490px;
-    left: 30px;
-    opacity: 85%;
+.bannerCopy {
+  position: absolute;
+  top: 28%;
+  color: var(--grey);
+  font-family: monospace;
+  margin-left: 20px;
 }
 
-/* Former Pro Musician | Aspiring MLE Line Header text */
-.subHeaderPosition {
-    position: relative;
-    display: block;
-    width: 900px;
-    top: -470px;
-    left: 35px;
-    opacity: 85%;
+.title {
+  font-size: 48px;
 }
 
+.subtitle {
+  font-size: 24px;
+  margin-top: 12px;
+}
+
+.socialLinks {
+  display: inline-block;
+  position: absolute;
+  top: 8%;
+  right: 4%;
+  width: 64px;
+}
+
+.socialLink {
+  width: 48px;
+}
+@media (min-width: 900px) {
+  .socialLink {
+    width: 64px;
+  }
+}
 
 .gitHubPosition {
-    position: relative;
-    display: block;
-    width: 65px;
-    top: -790px;
-    left: 1200px;
-    opacity: 85%;
+  /* make these icons instead of pngs. Icons are easy to style and are a smaller filesize. */
+  opacity: 85%;
+  margin-bottom: 24px;
 }
 
 .linkedInPosition {
-    position: relative;
-    display: block;
-    /* height: 50px; I don't understand why the size of the image is not changing from the css sheet. solved with inline html style addition*/
-    top: -760px;
-    left: 1205px;
-    opacity: 80%;
 }
 
-h1{
-    font-family: 'Courier New', Courier, monospace;
-    color: #2da4af;
-    font-weight: 100;
-    font-size: large;
-    margin-left: 20px;
-    padding-top: 10px;
+.mainBodyContainer {
+  margin: 24px auto 0;
+  padding: 8px;
+  font-family: 'Courier New', Courier, monospace;
+  color: var(--white);
+  font-size: 16px;
 }
 
-
-/* FIND A BETTER SOLUTION FOR ME */
-p, ul{ 
-    font-family: 'Courier New', Courier, monospace;
-    margin-left: 100px;
-    color: #FFFFFF;
-    font-size: small;
-    text-align: justify;
-    max-width: 1100px;
+.mainBodySection {
+  margin-bottom: 36px;
 }
 
+.mainBodySection h1 {
+  font-family: 'Courier New', Courier, monospace;
+  color: var(--header);
+  font-weight: 100;
+  font-size: 24px;
+  margin-left: max(3%, 30px);
+  margin-bottom: 20px;
+}
 
-.future_float_container {
-    display: flex;
-    margin-top: 45px;
-    height: 560px;
+.mainBodySection p {
+  margin: 12px 0;
+  margin-left: max(8%, 40px);
 }
 
-.future_float_child_L {
-    width: 26%;
-    float: left;
-    padding: 10px;
+.mainBodySection ul {
+  margin-left: max(14%, 50px);
 }
 
-.future_float_child_R {
-    position:relative;
-    width: 74%;
-    float: right;
-    padding: 10px;
+.futureContainer {
+  position: relative;
+  margin-bottom: 20px;
+  /* this is a hack because this div was not behaving correctly */
+  border: 1px solid rgba(0, 0, 0, 0);
+  height: min(500px, calc(100vh - 50%));
```

Comment:
This calculates how large the container should be, and as a result, the photo can only grow so large.